packages/typescript-estree/package.json

```diff
@@ -52,7 +52,7 @@
   },
   "devDependencies": {
     "@babel/code-frame": "^7.12.11",
-    "@babel/parser": "^7.12.11",
+    "@babel/parser": "^7.12.14",
```

Comment:
7.12.14 has issue with abstract methods in abstract classes, can you bump it to 7.12.15

Reply:
done :+1: fd6d72b